x-pack/plugins/observability_solution/observability_ai_assistant/server/service/chat_function_client/index.ts

```diff
@@ -120,13 +124,18 @@ export class ChatFunctionClient {
 
   getFunctions({
     filter,
-    scope,
+    scopes,
+    requiredFunctions,
   }: {
     filter?: string;
-    scope?: AssistantScope;
+    scopes?: AssistantScope[];
+    requiredFunctions?: string[];
```

Comment:
I'm not sure if I get this one, what is it for?

Reply:
For screencontexts that include actions that list a specific function as required. We send a list of those required functions so we don't leave them out if the screencontext requires it but we're in a different scope.